packages/env/lib/commands/destroy.js

```diff
@@ -0,0 +1,70 @@
+/**
+ * External dependencies
+ */
+const dockerCompose = require( 'docker-compose' );
+const util = require( 'util' );
+const fs = require( 'fs' ).promises;
+const inquirer = require( 'inquirer' );
+
+/**
+ * Promisified dependencies
+ */
+const rimraf = util.promisify( require( 'rimraf' ) );
+
+/**
+ * Internal dependencies
+ */
+const initConfig = require( '../init-config' );
+const stop = require( './stop' );
+
+/**
+ * Destroy the development server.
+ *
+ * @param {Object}  options
+ * @param {Object}  options.spinner A CLI spinner which indicates progress.
+ * @param {boolean} options.debug   True if debug mode is enabled.
+ */
+module.exports = async function destroy( { spinner, debug } ) {
+	const config = await initConfig( { spinner, debug } );
```

Comment:
Interestingly, `initConfig` actually creates and updates the docker compose file in the location. This means that when you check to see if the path exists, it will always exist even after you delete it.

So if I run `wp-env destroy` once, it does correctly delete everything. The problem is that if I run it again, it won't tell me that there are no files to remove. It'll just go through and remove everything again.

this should do the trick instead:

```suggestion
	const configPath = path.resolve( '.wp-env.json' );
	const { dockerComposeConfigPath, workDirectoryPath } = await readConfig( configPath );
```

Reply:
Thanks!!!!